Lab 5

Koshruk opened the pull request:
lab05

Koshruk committed:
added lines after navigation links in header

Koshruk committed:
added text at the botom of images in image section

Koshruk committed:
added sliding container on figure in portfolio
(fixes related to sizes needed)

Koshruk committed:
added sliding containers to all cards on portfolio

Koshruk committed:
added modal window

Koshruk committed:
skip

Koshruk committed:
some fixes

Koshruk committed:
added animation on opening modal window

Koshruk committed:
added animation on closing modal window

Koshruk committed:
fixed modal animation

vozniuk-lntu reviewed (approved):
1. декор елемент в хедері на портфоліо
2. падінги секції на портфоліо
3. перевірити транзішн іконок
4. оверлдей появу через опасіті зробити

https://docs.google.com/document/d/1IGz60qj7l4Pfh4r7uTIfl5K8VIr4CiYA/edit?usp=sharing&ouid=104518963218417128491&rtpof=true&sd=true

vozniuk-lntu commented:
ссилкам додати href 

https://docs.google.com/document/d/1VK6M0LGyzXwYyn_X1yMwagge74CG5As_/edit?usp=sharing&ouid=104518963218417128491&rtpof=true&sd=true

Koshruk committed:
fixed decor in header
fixed svg animations in header
fixed padding in portfolio
added animation for modal overlay
added href to all links

Koshruk committed:
fixed some syntax errors

Koshruk committed:
fixed some syntax errors
